src/Models/Tag.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Astrogoat\Blog\Models;
+
+use Helix\Fabrick\Icon;
+use Helix\Lego\Models\Contracts\Sectionable;
+use Helix\Lego\Models\Model as LegoModel;
+use Helix\Lego\Models\Traits\HasSections;
+
+class Tag extends LegoModel implements Sectionable
+{
+    use HasSections;
+
+    protected $table = 'article_tags';
```

Comment:
Would it be worth making the 'tags' more generic and part of strata proper so we could tag other models in the future?